datadog_lambda/tracing.py

```diff
@@ -412,13 +412,26 @@ def create_inferred_span(event, context):
         elif event_source.equals(EventTypes.SNS):
             logger.debug("SNS event detected. Inferring a span")
             return create_inferred_span_from_sns_event(event, context)
+        elif event_source.equals(EventTypes.KINESIS):
```

Comment:
long term we might consider a null object pattern for event detection. Not necessary for this PR, just a suggestion

Reply:
Agreed. One of the reasons this was implemented like this is that the original trigger detection code handles some known events, but also allows "known-unknown" events that just have a raw string in the "source" or "event_source" key. I think the direction in which we're moving is a little bit of a departure from Postel's law. Thats not a bad thing, but we want to make sure we don't lose any functionality.